src/cmd/fusesoc.rs

```diff
@@ -0,0 +1,753 @@
+        if present_core_files[pkg].is_empty() {
+            generate_files.insert(
+                pkg.to_string(),
+                pkg_manifest_paths[pkg]
+                    .clone()
+                    .join(format!("{}.core", pkg)),
+            );
+
+            fuse_depend_string.insert(
+                pkg.to_string(),
+                get_fuse_depend_string(
+                    pkg,
+                    &srcs,
+                    vendor_string.to_string(),
+                    top,
+                    version_string.clone(),
+                ),
+            );
+        } else {
+            let mut index = 0;
+            if present_core_files[pkg].len() > 1 {
+                let mut msg = format!("Multiple `.core` files already present for {}.\n", pkg);
+                writeln!(
+                    msg,
+                    "Please pick a `.core` file to use for this dependency.\n"
+                )
+                .unwrap();
+                for i in 0..present_core_files[pkg].len() {
+                    let file_str =
+                        read_to_string(&present_core_files[pkg][index]).map_err(|cause| {
+                            Error::chain(
+                                format!(
+                                    "Cannot open .core file {:?}.",
+                                    &present_core_files[pkg][index]
+                                ),
+                                cause,
+                            )
+                        })?;
+
+                    let fuse_core = parse_fuse_file(
+                        file_str,
+                        present_core_files[pkg][i].display().to_string(),
+                    )?;
+                    writeln!(
+                        msg,
+                        "{}) {} : {}",
+                        i,
+                        present_core_files[pkg][i]
+                            .as_path()
+                            .strip_prefix(&pkg_manifest_paths[pkg])
+                            .unwrap()
+                            .display(),
+                        fuse_core.name
+                    )
+                    .unwrap();
+                }
+                println!("{}", msg);
+                // Let user resolve conflict if both stderr and stdin go to a TTY.
+                if atty::is(atty::Stream::Stderr) && atty::is(atty::Stream::Stdin) {
+                    index = {
+                        loop {
+                            eprint!("Enter a number or hit enter to abort: ");
+                            io::stdout().flush().unwrap();
+                            let mut buffer = String::new();
+                            io::stdin().read_line(&mut buffer).unwrap();
+                            if buffer.starts_with('\n') {
+                                break Err(Error::new(msg));
+                            }
+                            let choice = match buffer.trim().parse::<usize>() {
```

Comment:
We only support choosing one core file atm. That is perfectly fine. I wanted to make sure I understood that correctly.